keycloak-export/src/main/java/io/cloudtrust/keycloak/export/ImportExportUtils.java

```diff
@@ -43,9 +43,6 @@ public static RealmModel importRealm(KeycloakSession session, KeycloakApplicatio
         }
 
         RealmManager realmManager = new RealmManager(session);
-        if (keycloak != null) {
```

Comment:
I thought it was useful in case of import at startup... We will have to be careful if we need to use this feature in the future

Reply:
Stian did remove this line in the upstream (https://github.com/keycloak/keycloak/commit/b8881b8ea062afa2f1b5a613dd856900b73f2121), and the API is not valid any more --> I've removed this; let's now see how it behaves :-)